src/OptimisticTokenVotingPlugin.sol

```diff
@@ -184,13 +185,18 @@ contract OptimisticTokenVotingPlugin is
     }
 
     /// @inheritdoc IOptimisticTokenVoting
+
     function totalVotingPower(uint256 _timestamp) public view returns (uint256) {
         return votingToken.getPastTotalSupply(_timestamp);
+        // Q(danielw): we should probably need to remove votes of `taikoL1` which holds a lot of TAIKO as bond deposits.
```

Comment:
If this balance is unrelated to L2 being up or not (effectiveVotingPower) then sure, let's update the expression.

Reply:
The balance of TaikoL1 is unrelated to the state of L2.